### ISSUE TITLE
The Loss in the final train is between 1 and 2, is it normal?

### ISSUE THREAD

**Issue by FlyEgle** [biubug6/Pytorch_Retinaface#123]
![image](https://user-images.githubusercontent.com/26165705/93156344-892fda80-f73a-11ea-9a40-d150a1b7a076.png)
